frontend/src/v5/ui/routes/dashboard/projects/calibration/calibration.component.tsx

```diff
@@ -0,0 +1,95 @@
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { generatePath, useParams } from 'react-router';
+import { Loader } from '@/v4/routes/components/loader/loader.component';
+import { DrawingsHooksSelectors } from '@/v5/services/selectorsHooks';
+import { FormattedMessage } from 'react-intl';
+import { useContainersData } from '../containers/containers.hooks';
+import { useFederationsData } from '../federations/federations.hooks';
+import { CalibrationParams, DRAWINGS_ROUTE } from '../../../routes.constants';
+import { EmptyPageView } from '@components/shared/emptyPageView/emptyPageView.styles';
+import { CalibrationStep } from './calibrationSteps/calibrationStep.component';
+import { Content } from '../projectContent/projectContent.styles';
+import { CalibrationContextComponent } from './calibrationContext';
+import { CalibrationHeader } from './calibrationHeader/calibrationHeader.component';
+import { Button } from '@controls/button';
+import { Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import { DrawingsActionsDispatchers } from '@/v5/services/actionsDispatchers';
+
+export const Calibration = () => {
+	const { teamspace, project, drawing } = useParams<CalibrationParams>();
+
+	const selectedDrawing = DrawingsHooksSelectors.selectDrawingById(drawing);
+	const { isListPending: containersArePending, containers } = useContainersData();
+	const { isListPending: federationsArePending, federations } = useFederationsData();
+	const isLoadingModels = containersArePending || federationsArePending;
+	const isLoadingDrawings = DrawingsHooksSelectors.selectIsListPending();
+	const hasModels = !![...containers, ...federations].length;
+
+	useEffect(() => {
+		if (selectedDrawing || !teamspace || !project) return;
+		DrawingsActionsDispatchers.fetchDrawings(teamspace, project);
+	}, [teamspace, project]);
+
+	if (isLoadingModels || isLoadingDrawings) return (<Loader />);
+
+	if (!hasModels) {
+		return (
+			<Content>
+				<EmptyPageView>
+					<FormattedMessage
+						id="calibration.emptyModels"
+						defaultMessage="This project is empty. Please, proceed to create a container or a federation to access this content."
+					/>
+				</EmptyPageView>
+			</Content>
+		);
+	}
+
+	if (!selectedDrawing) {
+		return (
+			<Content>
+				<EmptyPageView>
+					<FormattedMessage
+						id="calibration.invalidDrawing"
+						defaultMessage="The selected drawing was not found. Please, go back to the drawings list select the drawing to calibrate from there."
+					/>
+					<br />
+					<Button variant="contained">
+						<Link to={generatePath(DRAWINGS_ROUTE, { teamspace, project })}>
+							<FormattedMessage
+								id="calibration.invalidDrawing.goToDrawings"
+								defaultMessage="Open drawings"
+							/>
+						</Link>
+					</Button>
+				</EmptyPageView>
+			</Content>
+		);
+	}
+
+	return (
+		<CalibrationContextComponent>
+			<CalibrationHeader />
+			<Content>
+				<CalibrationStep />
```

Comment:
I dont see mockup content when the step changes.
<img width="967" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/deba7834-8af0-4693-bf3c-94c92035f564">

The idea is that next dev can replace the content in the mockup components and make them functional